src/App.js

```diff
@@ -1,13 +1,85 @@
 import React from 'react';
+import TodoList from "./components/TodoComponents/TodoList"
+import TodoForm from "./components/TodoComponents/TodoForm"
+import "./components/TodoComponents/Todo.css"
+
+const games = [
+    {
+      name: "Animal Crossing",
+      id: 1,
+      completed: false
+    },
+    {
+      name: "Dark Souls",
+      id: 2,
+      completed: false
+    },
+    {
+      name: "Rainbow 6 Siege",
+      id: 3,
+      completed: false
+    },
+    {
+      name: "Skyrim",
+      id: 4,
+      completed: false
+    },
+]
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  
+  constructor() {
+    super();
+    this.state = {
+      games
+    };
+  }
+
+  toggleGame = (gameId) => {
+    console.log(gameId)
+    this.setState({
+      games: this.state.games.map(game => {
+        if (game.id === gameId) {
+          return { ...game, completed: !game.completed }
+        }
+        return game
+      })
+    })
+  }
+
+  addGame = (e, gameName) => {
+    e.preventDefault();
+    
+    if (gameName.length > 0) {
+      const newGame = {
+        name: gameName,
+        id: Date.now(),
+        completed: false
+      };
+      console.log(newGame)
+      this.setState({
+        games: [...this.state.games, newGame]
+      });
+    }
+  }
+
+    clearGames = (e) => {
```

Comment:
Naming conventions are more important than they seem especially for someone reviewing code. Once you have to read someone else's code you'll understand. I mention it hear at `clearGames` but it applies to `toggleGame` because from the outside looking in it s hard to see exactly what these functions are doing. By this I mean that I have no clue what `toggleGame` is doing and I would expect `clearGames` to return an empty list of games. Alternatively, `toggleGameCompleteness` or `toggleGameCompletFlag` tell me a lot more about what the function is doing and `clearCompletedGames` is doing what I expect it to. Don't be scared of long names! They give you finger muscles!